src/kakarot/accounts/contract/library.cairo

```diff
@@ -42,13 +38,6 @@ func is_initialized_() -> (res: felt) {
 func evm_address() -> (evm_address: felt) {
 }
 
-// An event emitted whenever kakarot deploys a evm contract
-// evm_contract_address is the representation of the evm address of the contract
-// starknet_contract_address if the starknet address of the contract
-@event
```

Comment:
since we delete the event emitted on account creation, how does one retrieve starknet_address / evm_address of a newly deployed smart contract?

Reply:
good question. Actually this we unsued so we'd need to emit an event at some point, this is more related to https://github.com/sayajin-labs/kakarot/issues/491 ; I'm adding a comment there not to forget to emit

Reply:
I checked it, the internal calls of the create_account function return the deployed contract address

Reply:
And even without it we can compute the address and verify if it exists